configuration/azure/appconfig/metadata.yaml

```diff
@@ -0,0 +1,104 @@
+capabilities: []
+authenticationProfiles:
+  - title: "Connection string"
+    description: "Authenticate using a connection string."
+    metadata:
+      - name: connectionString
+        required: true
+        sensitive: true
+        description: "The Service Bus connection string."
+        example: '"BlobEndpoint=https://storagesample.blob.core.windows.net;SharedAccessSignature={KeySig}"'
+        # If omitted, uses the same values as "<root>.binding"
+        binding:
+          output: true
+          input: true
+  - title: "Azure AD: Managed identity"
+    description: "Authenticate using Azure AD and a managed identity."
+    metadata:
+      - name: azureClientId
+        required: false
+        description: "Client ID (application ID). Required if the service has multiple identities assigned."
+        example: '"c7dd251f-811f-4ba2-a905-acd4d3f8f08b"'
+  - title: "Azure AD: Client credentials"
+    description: "Authenticate using Azure AD with client credentials, also known as \"service principals\"."
+    metadata:
+      - name: azureTenantId
+        required: true
+        description: "ID of the Azure AD tenant"
+        example: '"cd4b2887-304c-47e1-b4d5-65447fdd542a"'
+      - name: azureClientId
+        required: true
+        description: "Client ID (application ID)"
+        example: '"c7dd251f-811f-4ba2-a905-acd4d3f8f08b"'
+      - name: azureClientSecret
+        sensitive: true
+        required: true
+        description: "Client secret (application password)"
+        example: '"Ecy3XG7zVZK3/vl/a2NSB+a1zXLa8RnMum/IgD0E"'
+  - title: "Azure AD: Client certificate"
+    description: "Authenticate using Azure AD with a client certificate (in PFX/PKCS#12 format). One of azureCertificate and azureCertificateFile is required."
+    metadata:
+      - name: azureTenantId
+        required: true
+        description: "ID of the Azure AD tenant."
+        example: '"cd4b2887-304c-47e1-b4d5-65447fdd542b"'
+      - name: azureClientId
+        required: true
+        description: "Client ID (application ID)."
+        example: '"c7dd251f-811f-4ba2-a905-acd4d3f8f08b"'
+      - name: azureCertificate
+        sensitive: true
+        required: false
+        description: "Certificate and private key (in PFX/PKCS#12 format)."
+        example: |
+          "-----BEGIN PRIVATE KEY-----\n MIIEvgI... \n -----END PRIVATE KEY----- \n -----BEGIN CERTIFICATE----- \n MIICoTC... \n -----END CERTIFICATE-----"
+      - name: azureCertificateFile
+        sensitive: true
+        required: false
+        description: "Path to PFX/PKCS#12 file on disk, containing the certificate and private key."
+        example: "/path/to/file.pem"
+      - name: azureCertificatePassword
+        sensitive: true
+        required: false
+        description: "Password for the certificate if encrypted."
+        example: "password"
+metadata:
+  - name: host
```

Comment:
Since this is only used with Azure AD, move this to the "azuread" auth profile. See exmaples of Service Bus